fem/fespace.cpp

```diff
@@ -3529,32 +3531,32 @@ FiniteElementCollection *FiniteElementSpace::Load(Mesh *m, std::istream &input)
             MFEM_VERIFY(nurbs_fec,
                         buff << ": NURBS FE collection is required!");
             MFEM_VERIFY(m->NURBSext, buff << ": NURBS mesh is required!");
-            MFEM_VERIFY(!NURBSext, buff << ": order redefinition!");
+            MFEM_VERIFY(!nurbs_ext, buff << ": order redefinition!");
             if (buff == "NURBS_order")
             {
                int order;
                input >> order;
-               NURBSext = new NURBSExtension(m->NURBSext, order);
+               nurbs_ext = new NURBSExtension(m->NURBSext, order);
             }
             else
             {
                Array<int> orders;
                orders.Load(m->NURBSext->GetNKV(), input);
-               NURBSext = new NURBSExtension(m->NURBSext, orders);
+               nurbs_ext = new NURBSExtension(m->NURBSext, orders);
             }
          }
          else if (buff == "NURBS_periodic")
          {
             Array<int> master, slave;
             master.Load(input);
             slave.Load(input);
-            NURBSext->ConnectBoundaries(master,slave);
+            nurbs_ext->ConnectBoundaries(master,slave);
          }
          else if (buff == "NURBS_weights")
          {
-            MFEM_VERIFY(NURBSext, "NURBS_weights: NURBS_orders have to be "
+            MFEM_VERIFY(nurbs_ext, "NURBS_weights: NURBS_orders have to be "
                         "specified before NURBS_weights!");
-            NURBSext->GetWeights().Load(input, NURBSext->GetNDof());
+            nurbs_ext->GetWeights().Load(input, NURBSext->GetNDof());
```

Comment:
`NURBSext->GetNDof()` seems this be changed to `nurbs_ext`

I was starting to feel restless that I won't find an actual error.

Reply:
Good catch!